mkdocs.yml

```diff
@@ -89,6 +89,7 @@ nav:
             - './microservices/application/AdvancedTopics.md'
         - './microservices/application/AppServiceConfigurable.md'
         - './microservices/application/GeneralAppServiceConfig.md'
+        - './microservices/application/V2Migration.md'
```

Comment:
Are lines `90-92` same level as `80-83`? Is there a reason they don't have titles, ala:

```yaml
        - App Service Configurable: './microservices/application/AppServiceConfigurable.md'
        - General App Service Config: './microservices/application/GeneralAppServiceConfig.md'
        - V2 Migration: './microservices/application/V2Migration.md'
```

Reply:
Yes, they are at the same level. 
makedocs pulls the title from the first header in the file if the title is not supplied here. The ones supplied here are shorter versions or different than in the file.

Reply:
![image](https://user-images.githubusercontent.com/44779287/125351691-3e103500-e315-11eb-9d4d-6cee5253ba6b.png)